kalite/topic_tools/tests/helper_tests.py

```diff
@@ -11,74 +11,71 @@
-		self.original_exerciselog2.latest_activity_timestamp = self.TIMESTAMP_EARLY
-		self.original_exerciselog2.completion_timestamp = self.TIMESTAMP_EARLY
-		self.original_exerciselog2.struggling = False
-		self.original_exerciselog2.save()
+        self.user1 = FacilityUser(username=self.USERNAME1, facility=self.facility)
+        self.user1.set_password(self.PASSWORD)
+        self.user1.save()
 
-		self.original_exerciselog2 = ExerciseLog(exercise_id=self.EXERCISE_ID2, user = self.user1)
-		self.original_exerciselog2.points = self.ORIGINAL_POINTS
-		self.original_exerciselog2.attempts = self.ORIGINAL_POINTS
-		self.original_exerciselog2.streak_progress = self.ORIGINAL_STREAK_PROGRESS
-		self.original_exerciselog2.latest_activity_timestamp = self.TIMESTAMP_LATER
-		self.original_exerciselog2.completion_timestamp = self.TIMESTAMP_LATER
-		self.original_exerciselog2.struggling = False
-		self.original_exerciselog2.save()
+        # insert some exercise activity
+        self.original_exerciselog2 = ExerciseLog(exercise_id=self.EXERCISE_ID, user=self.user1)
+        self.original_exerciselog2.points = self.ORIGINAL_POINTS
+        self.original_exerciselog2.attempts = self.ORIGINAL_POINTS
+        self.original_exerciselog2.streak_progress = self.ORIGINAL_STREAK_PROGRESS
+        self.original_exerciselog2.latest_activity_timestamp = self.TIMESTAMP_EARLY
+        self.original_exerciselog2.completion_timestamp = self.TIMESTAMP_EARLY
+        self.original_exerciselog2.struggling = False
+        self.original_exerciselog2.save()
 
-	def tearDown(self):
-		'''Performed after each test'''
-		self.user_with_activity = None
-		self.user_with_no_activity = None
+        self.original_exerciselog2 = ExerciseLog(exercise_id=self.EXERCISE_ID2, user=self.user1)
+        self.original_exerciselog2.points = self.ORIGINAL_POINTS
+        self.original_exerciselog2.attempts = self.ORIGINAL_POINTS
+        self.original_exerciselog2.streak_progress = self.ORIGINAL_STREAK_PROGRESS
+        self.original_exerciselog2.latest_activity_timestamp = self.TIMESTAMP_LATER
+        self.original_exerciselog2.completion_timestamp = self.TIMESTAMP_LATER
+        self.original_exerciselog2.struggling = False
+        self.original_exerciselog2.save()
 
-	def test_exercises_from_topics(self):
-		'''get_exercises_from_topics()'''
+    def tearDown(self):
+        '''Performed after each test'''
+        self.user_with_activity = None
+        self.user_with_no_activity = None
 
-		expected = ['converting_fractions_to_decimals',
-					'converting_decimals_to_fractions_1', 
-					'converting_decimals_to_fractions_2']
-		actual = get_exercises_from_topics([self.TOPIC_TO_TEST])
+    def test_exercises_from_topics(self):
+        '''get_exercises_from_topics()'''
 
-		self.assertEqual(expected, actual)
+        expected = [u'counting-out-1-20-objects', u'counting-objects', u'one-more--one-less']
+        actual = get_exercises_from_topics(['cc-early-math-counting'])
```

Comment:
@rtibbles the diff is messed up because I converted tabs to spaces, but this is the essence: I changed a non-existing or empty topic ID to one that existed and put its Exercise id's in place of the old ones.